internal/controller/route_common_test.go

```diff
@@ -51,6 +52,23 @@ func (s *RouteTestSuite) Test_GeneratesExpectedRoutes() {
 	}
 }
 
+func (s *RouteTestSuite) Test_GeneratesExpectedPaths() {
+	routes := s.generateRoutes()
+	actualPaths := make(map[string]int)
+
+	for _, r := range routes {
+		actualPaths[r.Path()]++
+	}
+
+	s.Require().Equal(len(s.expectedPaths), len(actualPaths))
+	for expectedPath, expectedCount := range s.expectedPaths {
```

Comment:
We now assert the generated paths on top of the methods. This will help to control that everything still works when we will migrate the `galactic-sovereign-service` routes.